README.md

```diff
@@ -1,6 +1,6 @@
 # .NET Source-Build
 
-[![Join the chat at https://gitter.im/dotnet/source-build](https://badges.gitter.im/dotnet/source-build.svg)](https://gitter.im/dotnet/source-build?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+We now use [GitHub discussions](https://github.com/dotnet/source-build/discussions) for announcements, questions, suggestions, and sharing information with other members of the source-build community.  Please join in!
```

Comment:
I prefer avoiding point in time verbiage - "We now use".  It adds negative value to readers who don't have the historical context.

```suggestion
Please use [GitHub discussions](https://github.com/dotnet/source-build/discussions) to see announcements, ask questions, make suggestions, and share information with other members of the source-build community.
```